be/src/vec/exec/scan/pip_scanner_context.h

```diff
@@ -152,23 +152,10 @@ class PipScannerContext : public vectorized::ScannerContext {
     }
 
     bool has_enough_space_in_blocks_queue() const override {
-        return _current_used_bytes < _max_bytes_in_queue / 2 * _num_parallel_instances;
+        return _current_used_bytes < _max_bytes_in_queue  * _num_parallel_instances;
```

Comment:
why not using scan_queue_mem_limit?

Reply:
_max_bytes_in_queue  is the scan_queue_mem_limit